modules/platform/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoRequestControllerFilter.java

```diff
@@ -70,13 +70,15 @@
 
     protected static final int LOCK_TIMEOUT_S = 120;
 
+    protected IdempotentRequestHandler handler;
```

Comment:
`handler` -> `idempotentRequestHandler`

Reply:
As discussed, moved this handler to a dedicated filter class.